attributes/wsus_client.rb

```diff
@@ -43,7 +43,7 @@
 # => :download  = Download updates, but lets users install them
 # => :install   = Download and install updates
 # => :manual    = Lets the users configure the behavior
-default['wsus_client']['automatic_update_behavior']                = :download
+default['wsus_client']['automatic_update_behavior']                = :install
```

Comment:
Are we managing blast radius through policy files (environments)?

Reply:
Sorta. The scope of nodes would be managed via the `default['wsus_client']['update_group']` attribute.